torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -336,6 +336,33 @@ void Reducer::check_grad_layout(
   }
 }
 
+void Reducer::point_grads_to_bucket() {
+  std::unordered_set<int> zeroed_buckets;
+  for (size_t variable_index = 0; variable_index < params_.size();
+       ++variable_index) {
+    const auto& bucket_index = variable_locators_[variable_index];
```

Comment:
nit:
```suggestion
    const auto& variable_locator = variable_locators_[variable_index];
```

`variable_locator.bucket_index` may be easier to understand than `bucket_index.bucket_index`, which may be a misnomer.